### PR TITLE
docs: wrap core demos with a default fieldgroup

### PR DESCRIPTION
**Pull request by sebald**

### CODE REVIEW

File: docs/ui/ComponentDemo.tsx [hunk 1]
```diff
@@ -35,6 +35,13 @@ export const ComponentDemo = ({ name, children }: ComponentDemoProps) => {
   const Demo = registry[name].demo;
   const { current, themes } = useThemeSwitch();
 
+  const Wrapper = ({ children }: { children: ReactNode }) =>
+    current === 'core' ? (
+      <FieldGroup labelWidth="100px">{children}</FieldGroup>
```
Comment (sarahgm) [inline docs/ui/ComponentDemo.tsx:40]: in some demos are currently `<FieldGroups>` around, can you remove them than? :)

Comment (sebald) [reply docs/ui/ComponentDemo.tsx:40]: Should we? I thought this can be the default, but maybe, if you need more than 100px you add them in the demo too :)

Comment (sarahgm) [reply docs/ui/ComponentDemo.tsx:40]: ok I understand wasn't sure if its possible to have a FieldGroup in a FieldGroup

Comment (sebald) [reply docs/ui/ComponentDemo.tsx:40]: should be since it is just a context :)

**Review by sarahgm — APPROVED** [review]

**Pull request merged**